core/dbt/contracts/graph/manifest.py

```diff
@@ -992,6 +1005,25 @@ def pydantic_semantic_manifest(self) -> PydanticSemanticManifest:
         for metric in self.metrics.values():
             pydantic_semantic_manifest.metrics.append(PydanticMetric.parse_obj(metric.to_dict()))
 
+        # Look for time-spine table model and create time spine table configuration
+        if self.semantic_models:
+            # Get model for time_spine_table
+            time_spine_model_name = "metricflow_time_spine"
+            model = self.ref_lookup.find(time_spine_model_name, None, None, self)
+            if not model:
+                raise ParsingError(
+                    "Semantic models require a 'metricflow_time_spine' model. See Metricflow docs."
```

Comment:
What do you think about something like?
```suggestion
                    "The semantic layer requires a 'metricflow_time_spine' model in the project, but none was found. Guidance on creating this model can be found on our docs site (https://docs.getdbt.com/docs/build/metricflow-time-spine)"
```